cron/logger_test.bash

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+base_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd -P )"
+source "$base_dir"/logger.bash
+configure_logger logger_test.bash
+
+trap_exit() {
```

Comment:
Might be good to have this in `logger.bash` rather than copying it into every script